processor/metricstatsprocessor/README.md

```diff
@@ -8,27 +8,27 @@ This processor aggregates metrics over a configurable interval, allowing for met
 - Metrics
 
 ## How it works
-1. The user configures the aggregation processor in the desired metrics pipeline.
+1. The user configures the metricstats processor in the desired metrics pipeline.
 2. Every metric that flows through the pipeline is matched against the provided `include` regex.
 3. If the metric name does not match the `include` regex, the metric passes through the processor.
 4. If the metric matches, but is not a gauge or cumulative sum, the metric passes through the processor.
-5. If the metric name does match, and the metric is a gauge or cumulative sum, the metric is added to an aggregate based on its attributes. The metric does not continue down the pipeline.
-6. After the configured `interval` has passed, all aggregate metrics are emitted. Aggregate metrics are emitted with "${metric_name}.${aggregation}" e.g. if you take the average of the metric `system.cpu.utilization`, the aggregated metric would be `system.cpu.utilization.avg`.
-7. All aggregations are cleared, and will not be emitted on the next interval, unless another matching metric enters the pipeline.
+5. If the metric name does match, and the metric is a gauge or cumulative sum, the metric is added to a statistic based on its attributes. The metric does not continue down the pipeline.
+6. After the configured `interval` has passed, all calculated metrics are emitted. Calculated metrics are emitted with a name of "${metric_name}.${statistic_type}" e.g. if you take the average of the metric `system.cpu.utilization`, the calculated metric would be `system.cpu.utilization.avg`.
```

Comment:
Not sure if this is an issue of my VS code markdown preview, but it doesnt like 
```
"${metric_name}.${statistic_type}"
```
![image](https://user-images.githubusercontent.com/64915094/220158765-1a714003-3850-4362-89f9-881b0f4c30a2.png)